recipes-mender/mender/mender_%.bbappend

```diff
@@ -0,0 +1,25 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+MENDER_SERVER_URL = "dummy"
+
+SRC_URI += " \
+    file://mender-restore-saved-settings.sh \
+"
+
+FILES_${PN} += " \
+     ${sbindir}/mender-restore-saved-settings.sh \
+"
+
+do_install_prepend() {
+    # Update the service file to invoke mender-generate-certificate
+    cat >> ${WORKDIR}/${SYSTEMD_SERVICE_${PN}} <<-EOF
+
+	[Service]
+	ExecStartPre=${sbindir}/mender-restore-saved-settings.sh
```

Comment:
Maybe I'm not understanding the full picture, but how is this going to help after an OTA update? This is only included in the toradex image we produce, so any OTA update you make afterwards will receive the token through the normal Yocto configuration variables, and not through here, right?

Reply:
Well, since we are providing a prebuilt binary for initial provisioning with the Easy Installer, I feel like we should have a prebuilt artifact that people can use to get a feel for the workflow.  Obviously they will eventually want to create their own which should contain their token but as a getting started step, it would be nice to have a generic one available.  Without making these settings persist, we obviously cannot do that.

Toradex has already asked for an artifact for testing purposes.

Reply:
Ok, I see now.